public/layouts/dashboard/dashboard-user.html

```diff
@@ -34,8 +34,21 @@
         </li>
 
         <li class="dropdown-toolbar__item" ng-if="subscriptionsSupported">
-            <a class="dropdown-toolbar__item-label" href="#" ng-click="registerSubscription()">
-                <span class="dropdown-toolbar__item-title" title="">Subscribe to this page</span>
+            <span class="dropdown-toolbar__item-label" href="#" ng-click="registerSubscription()" ng-if="subscriptionStatus">
```

Comment:
the semantic web is 😢

Reply:
Oops it shouldn't have the `ng-click` handler if it's currently subscribing